msl_expressions/autogenerated/src/Plans/GenericStandards/StandardAlignToPoint.cpp

```diff
@@ -139,17 +140,12 @@ namespace alica
             this->setSuccess(true);
         }
 
-        if (!std::isnan(mc.motion.translation))
-        {
-            send(mc);
-        }
-
-
+        send(mc);
```

Comment:
why is the nan check removed ?
robotmovement is used therefore a nan checkshould be done